recipe/meta.yaml

```
@@ -16,7 +16,7 @@ requirements:
   run:
     - cuda-command-line-tools {{ version }}
     - cuda-visual-tools {{ version }}
-    - gds-tools 1.8.1.2                      # [linux64]
+    - gds-tools 1.9.0.20                      # [linux64]
```

Shouldn't we have this for `linux_aarch64` too?

```suggestion
    - gds-tools 1.9.0.20                      # [linux64 or aarch64]
```

---

Yes, thanks!